releases/release-1.34/README.md

```diff
@@ -0,0 +1,110 @@
+| Release Highlights deadline                                                            | Comms                         | Friday 1st August 2025                                                                                          | week 11  |                                                                          |
+| Burndown Meetings daily (Tuesday & Thursday over Slack)                                | Lead                          | Monday 4th August 2025                                                                                          | week 12  |                                                                          |
+| release-1.34 branch created                                                            | Branch Manager                | Wednesday 6th August 2025                                                                                       | week 12  |                                                                          |
+| release-1.34 jobs created                                                              | Branch Manager                | Wednesday 6th August 2025                                                                                       | week 12  |                                                                          |
+| Start final draft of Release Notes                                                     | Docs Lead                     | Wednesday 6th August 2025                                                                                       | week 12  |                                                                          |
+| **[Docs Freeze]**                                                                      | Docs Lead                     | Wednesday 6th August 2025                                                                                       | week 12  |                                                                          |
+| 1.34.0-rc.0 released                                                                   | Branch Manager                | Wednesday 6th August 2025                                                                                       | week 12  | [1.34-informing], [1.34-blocking], [master-blocking], [master-informing] |
+| Feature blogs ready to review                                                          | Enhancement Owner / SIG Leads | Friday 8th August 2025                                                                                          | week 12  |                                                                          |
+| Release blog ready to review                                                           | Comms / Docs                  | [02:00 UTC Friday 15th August 2025 / 19:00 PDT Thursday 14th August 2025](https://everytimezone.com/s/815eefb6) | week 13  |                                                                          |
```

Comment:
`Feature blogs ready to review` deadline is two days after the docs freeze - as discussed in the v1.33 retro. 
Also, I've moved `Release blog ready to review` deadline to the next week to not overburden the comms team.